monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -0,0 +1,248 @@
+        self.data_stats_filename = data_stats_filename
+
+    def set_data_source(self, data_list_file):
+        self.data_list_file = data_list_file
+
+    def fill_template_config(self, data_stats_filename):
+        # self.algorithm_dir = os.path.join(self.output_path, "segresnet2d")
+        # self.inference_script = "scripts/infer.py"
+        pass
+
+    def export_to_disk(self, output_path, algo_name, **kwargs):
+        self.fill_template_config(self.data_stats_filename)
+        write_path = os.path.join(output_path, algo_name)
+        self.cfg["bundle_root"] = write_path
+        os.makedirs(write_path, exist_ok=True)
+        # handling scripts files
+        output_scripts_path = os.path.join(write_path, "scripts")
+        if os.path.exists(output_scripts_path):
+            shutil.rmtree(output_scripts_path)
+        if self.scripts_path is not None and os.path.exists(self.scripts_path):
+            shutil.copytree(self.scripts_path, output_scripts_path)
+        # handling config files
+        output_config_path = os.path.join(write_path, "configs")
+        if os.path.exists(output_config_path):
+            shutil.rmtree(output_config_path)
+        os.makedirs(output_config_path, exist_ok=True)
+        output_config_file = os.path.join(output_config_path, "algo_config.yaml")
+        ConfigParser.export_config_file(self.cfg.config, output_config_file, fmt="yaml", default_flow_style=None)
+        with open(output_config_file, "r+") as f:
+            lines = f.readlines()
+            f.seek(0)
+            f.write(f"# Generated automatically by `{__name__}`\n")
+            f.write("# For more information please visit: https://docs.monai.io/\n\n")
+            for item in ensure_tuple(self.template_configs):
+                f.write(f"# source file: {item}\n")
+            f.write("\n\n")
+            f.writelines(lines)
+        print(write_path)
+        self.output_path = write_path
+
+    def train(self):
+        pass
+
+    def get_score(self, *args, **kwargs):
+        pass
+
+
+class SegresnetAlgo(BundleAlgo):
+    def fill_template_config(self, data_stats_filename):
+        if data_stats_filename is None or not os.path.exists(str(data_stats_filename)):
+            return
+        data_cfg = ConfigParser(globals=False)
+        data_cfg.read_config(data_stats_filename)
+        patch_size = [320, 320]
+        max_shape = data_cfg["stats_summary#image_stats#shape#max"]
+        patch_size = [
+            max(32, shape_k // 32 * 32) if shape_k < p_k else p_k for p_k, shape_k in zip(patch_size, max_shape)
+        ]
+        self.cfg["patch_size#0"], self.cfg["patch_size#1"] = patch_size
+        self.cfg["patch_size_valid#0"], self.cfg["patch_size_valid#1"] = patch_size
+        data_src_cfg = ConfigParser(globals=False)
+        if self.data_list_file is not None and os.path.exists(str(self.data_list_file)):
+            data_src_cfg.read_config(self.data_list_file)
+            self.cfg.update(
```

Comment:
A simple verification may be considered here before the update. For example, modality needs to be "ct" and "mri". Some existing code may be referenced from Line 144-171 (https://github.com/mingxin-zheng/MONAI/blob/1e6b59f513976c61bcba293b0b84ce1f80e01b42/monai/apps/auto3d/algorithm_autoconfig.py#L144)